### PR TITLE
Update xopera-saas docs with the consent endpoint

### PR DESCRIPTION
**Pull request by sstanovnik**
This needs to be merged after releasing the new version of xopera-saas.

### CODE REVIEW

File: docs/saas.rst [hunk 1]
```diff
@@ -153,6 +153,9 @@ The following is a complete example of using the xOpera SaaS API through ``curl`
     # final login step, sets cookies and automatically completes the /secret request
     cookiecurl "$redirect_url" -d "username=$your_username" -d "password=$your_password" -d credentialId=""
 
+    # xopera-saas requires you to be mindful
```
Comment (anzoman) [inline docs/saas.rst:156]: This could be also mentioned somewhere in the documentation above.

Comment (sstanovnik) [reply docs/saas.rst:156]: Done! I've put it in the warnings section at the bottom, because it applies to all kinds of access to xopera-saas.

Comment (anzoman) [reply docs/saas.rst:156]: Great!

**Comment by sstanovnik** [comment]
A new version of `xopera-saas` has been released and it includes consent, so this is ready to be merged.

**Review by anzoman — APPROVED** [review]
Thanks sir @sstanovnik I approve your changes.

**Pull request merged**